### ISSUE TITLE
Renommage Graph

### ISSUE THREAD

**Issue by GrayCat911** [guillaumezin/DomoticzLinky#78]
Bonjour,

Est-il possible de renommer "Utilisation 1" et "Utilisation 2" en "Heure Creuse" et "Heure Pleine" ? Ou de donner la possibilité de nommer les graphes en fonction de notre besoin ?

**Comment by guillaumezin** [comment]
Non, ce n'est pas possible depuis le plugin, ça oblige à une refonte du code C++ et HTML de Domoticz. Chat échaudé craint l'eau froide, j'ai eu tellement de mal à faire accepter les précédents patchs par le mainteneur principale de Domoticz, pour que ce plugin voit le jour, que je ne veux plus m'y frotter. Désolé.
